app/Repositories/Tasks.ts

```diff
@@ -0,0 +1,20 @@
+import tasksArray from '../database/mocks/tasks.ts';
+import Task from '../Entities/Task.ts';
+
+class Tasks
```

Comment:
Acredito que ficará melhor se esse nome for destinado a uma interface, essa classe passar a se chamar `TasksMock` e implementar a interface. Também pode ser um boa ideia o array de dados ser passado pelo construtor.

Reply:
Concordo plenamente